web/js/roomController.js

```diff
@@ -699,12 +699,9 @@ PhoneNumberController, ResizeSensor, maxUsersPerRoom */
 
       if (!otHelper.isMyself(evt.from)) {
         _sharedStatus.roomMuted = muteAllSwitch;
-        if (muteAllSwitch) {
-          setAudioStatus(muteAllSwitch);
-          Utils.sendEvent('roomController:roomMuted', { isJoining: false });
-        } else {
-          RoomView.showConfirmChangeMicStatus(muteAllSwitch).then(setNewAudioStatus);
-        }
+         setAudioStatus(muteAllSwitch);
+         Utils.sendEvent('roomController:roomMuted', { isJoining: false });
```

Comment:
On clients that receive this message, I am seeing an error in the `setSwitchStatus()` method in roomView.js. `domElem` is undefined in this line:

```
    const oldStatus = domElem.classList.contains('activated');
```

Reply:
fixed in latest commit